app/assets/javascripts/components/sampleUpload.jsx

```diff
@@ -51,18 +51,20 @@ class SampleUpload extends React.Component {
       selectedJobQueue: this.selected.jobQueue || '',
```

Comment:
i know this wasn't part of this diff review but in line 23 of this file, there is a reference to hostGenomes[0] which breaks if there aren't any hostGenomes uploaded yet (e.g. when I do a ./bin/setup from scratch). could you please fix that, either as part of this diff, or separately?

Reply:
Sure would reproduce this error and apply a fix

Reply:
I think the solution would be to seed the database with a host genome as part of bin/setup. I can add that

Reply:
Yeah I have also at least prevented the page from breaking if there aren't any host genomes